### ISSUE TITLE
Openplanet fails to load certain characters

### ISSUE THREAD

**Issue by Fort-TM** [openplanet-nl/openplanet-issues#611]
When using Better Chat, Openplanet fails to load certain characters for some club tags

![image](https://github.com/user-attachments/assets/46f0ca5c-7681-4ce3-9f1d-743a1f22aa7a)

The clan tag in this case is `װℯⅴℴ`, with the last 3 not loading

Unicode characters are (I believe):

* U+212F
* U+2174
* U+2134

And here's the font loaded by Better Chat https://github.com/codecat/tm-better-chat/blob/67c1c591496df71d03fd715587913f6d0065b716/src/main.as#L80

**Comment by Fort-TM** [comment]
Some more:

![image](https://github.com/user-attachments/assets/04e95fe4-7a51-46c3-8149-57169a506230)

Mystral_'s clan is [ADHD](https://trackmania.io/#/clubs/54676), clan tag `ⓐⓓⓗⓓッ`

![image](https://github.com/user-attachments/assets/7f0cb9c6-72e1-48d4-9c25-e5d5e3f9b055)

Bloose's clan is [OCD](https://trackmania.io/#/clubs/34800), clan tag `ⓞⓒⓓッ`

**Comment by codecat** [comment]
I can't support all of these right now due to limitations in imgui's font loading.

**Comment by codecat** [comment]
Duplicate of #39

**Comment by Fort-TM** [comment]
I assume the underlying issue with Imgui is the same? Because the issues look nothing alike otherwise

**Comment by codecat** [comment]
Indeed. Once that issue is solved, we can (and will) add all Unicode characters.
